Services/CryptionService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using LethalCompany_Backend.Services.Interfaces;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace LethalCompany_Backend.Services;
+
+public class CryptionService : ICryptionService
+{
+    public string Decrypt(string key)
+    {
+        try
+        {
+            string _key = "abcdefgh";
+            string privatekey = "hgfedcba";
```

Comment:
You've put the key and private key in the .env in the react project. This is how it is supposed to be. Not in static code. put it in appsettings.